code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -3410,6 +3459,19 @@
 	M.drowsyness = 0
 	M.stuttering = 0
 	M.confused = 0
+//for the extremly drunk
+/datum/reagent/ethylredestroy
+	name = "Ethylredestroy"
+	id = ETHYLREDESTROY
+	description = "Removes all alcohol in someone's system with some minor hallucagens."
```

Comment:
Can you add a chemical heating version of Alkysine because I need some after reading this name and this description